apps/backend/src/modules/projects/project.entity.ts

```diff
@@ -9,40 +9,48 @@ class ProjectEntity implements Entity {
 
 	private lastActivityDate!: null | string;
 
+	private lastActivityUserName: null | string;
+
 	private name!: string;
 
 	private constructor({
 		description,
 		id,
 		lastActivityDate,
+		lastActivityUserName,
 		name,
 	}: {
 		description: string;
 		id: null | number;
 		lastActivityDate: null | string;
+		lastActivityUserName: null | string;
```

Comment:
last activity date is a column to track the last commit activity, not analytics sync, for analytics sync we need to save the date and time of sync and the user who is using the api key

```
analyticsLastSyncedAt: null | string;
analyticsLastSyncedByUserId: null | number
```

Reply:
I'm a bit confused about where this analyticsLastSyncedByUserId, analyticsLastSyncedAt data should come from

Reply:
you take user name correctly, analyticsLastSyncedAt should be the date this log was created. For user created it's better to return it with id:
```
analyticsLastSyncedByUser: null | {
  id: number,
  name: string 
}
```